docs/pages/connect-your-client/tsh.mdx

```diff
@@ -6,142 +6,244 @@ labels:
+</details>
 
-# `tsh ssh` takes the same arguments as the OpenSSH client:
-$ tsh ssh -o ForwardAgent=yes user@node
-$ tsh ssh -o AddKeysToAgent=yes user@node
+### Connect to a resource
 
-# You can even create a convenient symlink:
-$ ln -s /path/to/tsh /path/to/ssh
+Once you have determined a resource to connect to, the next step is to access
+the resource with `tsh`, which handles authentication to your Teleport cluster
+and routing traffic to and from local clients.
 
-# ... and now your 'ssh' command is calling Teleport's `tsh ssh`
-$ ssh user@host
+You can only connect to a Windows desktop using the Teleport Web UI or Teleport
+Connect. To do so, use the left sidebar to navigate to **Resources > Desktops**
+and select a desktop to connect to.
 
-# This command removes SSH certificates from a user's machine:
-$ tsh logout
+Select a resource type for instructions on connecting to the resource with
+`tsh`:
+
+<Tabs>
+<TabItem label="Servers">
+
+Run the `tsh ssh` command to connect to a server, specifying the login to assume
+on the server you are connecting to. The following command connects to the
+server `mynode` as user `root`:
+
+```code
+$ tsh ssh root@mynode
 ```
 
+`tsh ssh` takes the same arguments and flags as the OpenSSH client. For more
+information on connecting to servers, see [Connecting to SSH
+servers](#connecting-to-ssh-servers).
+
 </TabItem>
+<TabItem label="Applications">
 
-</Tabs>
+You can access web applications registered with Teleport through Teleport
+Connect and the Teleport Web UI. From the left sidebar, navigate to 
+**Resources > Applications** and choose an application to access. 
 
-In other words, Teleport was designed to be fully compatible with existing
-SSH-based workflows and does not require users to learn anything new, other than
-to call [`tsh login`](../reference/cli/tsh.mdx#tsh-login) in the beginning.
+For HTTP APIs protected with Teleport, use `tsh apps login` to receive a
+certificate authorized to access the application, which in this example, is
+`grafana`:
 
-## Installing tsh
+ ```code
+ $ tsh apps login grafana
+ ```
 
-Follow the instructions below to install the `tsh` binary.
+You can then access the application with a command like the following:
 
-We recommend installing `tsh` of the same major version as the version used in
-your Teleport cluster.
+```code
+$ curl \
+  --cert /Users/alice/.tsh/keys/teleport.example.com/alice-app/cluster-name/grafana-x509.pem \
+  --key /Users/alice/.tsh/keys/teleport.example.com/alice \
+  https://grafana.teleport.example.com:3080
+```
 
-To find the version number, either:
+For TCP applications, use `tsh` to start a local proxy server and connect to
+your application with your client of choice. This example connects to the app
+`tcp-app`:
 
-- In the Web UI, select your username in the upper right, then click
-  **Help & Support**. You will see the version of your Teleport
-  cluster under **CLUSTER INFORMATION**.
+```code
+$ tsh proxy app tcp-app
+```
```

Comment:
Both commands work for both types of apps. The only real difference is that the proxy automatically manages the certs and enriches with them connections made through the proxy whereas `tctl apps login` just fetches the cert that you have to them manually use with whichever client that you use.

I'd say in the vast majority of cases, a proxy is preferred, so I think we could first start with it and only later mention `tsh apps login`.

It'd be a good place to link to the VNet docs too since VNet makes this even easier than `tsh proxy app`.